src/components/SelectionList/BaseSelectionList.js

```diff
@@ -259,33 +273,40 @@ function BaseSelectionList({
 
     const renderItem = ({item, index, section}) => {
         const normalizedIndex = index + lodashGet(section, 'indexOffset', 0);
-        const isDisabled = section.isDisabled;
+        const isDisabled = section.isDisabled || item.isDisabled;
         const isItemFocused = !isDisabled && focusedIndex === normalizedIndex;
         // We only create tooltips for the first 10 users or so since some reports have hundreds of users, causing performance to degrade.
         const showTooltip = normalizedIndex < 10;
 
-        if (canSelectMultiple) {
-            return (
-                <UserListItem
-                    item={item}
-                    isFocused={isItemFocused}
-                    onSelectRow={() => selectRow(item, true)}
-                    onDismissError={onDismissError}
-                    showTooltip={showTooltip}
-                />
-            );
-        }
-
         return (
-            <RadioListItem
+            <BaseListItem
                 item={item}
                 isFocused={isItemFocused}
                 isDisabled={isDisabled}
+                showTooltip={showTooltip}
+                canSelectMultiple={canSelectMultiple}
                 onSelectRow={() => selectRow(item, true)}
+                onDismissError={onDismissError}
             />
         );
     };
 
+    const scrollToFocusedIndexOnFirstRender = useCallback(() => {
```

Comment:
It looks like introducing this caused issue: https://github.com/Expensify/App/issues/31660 (fixed by this PR: https://github.com/Expensify/App/pull/31737)